src/clients/storage_auto.rb

```diff
@@ -86,10 +89,25 @@ def main
       Builtins.y2milestone("Storage auto finished")
       Builtins.y2milestone("----------------------------------------")
 
-      deep_copy(@ret) 
+      deep_copy(@ret)
 
       # EOF
     end
+
+  private
+
+    # Returns a dialog to edit the storage
+    #
+    # It uses the current partition plan.
+    #
+    # @return [Y2Storage::Dialogs::Storage] Storage dialog
+    def build_storage_dialog
+      part_plan = AutoinstPartPlan.Export
+      # FIXME: workaround to avoid crashing the dialog
```

Comment:
Do we have a note somewhere (in any subsequent card) to improve this?

Reply:
I'll ask to @imobachgs 

If that note does not exists yet, I'll add it somewhere.